### PR TITLE
docs: serialization clarification

### PR DESCRIPTION
**Pull request by alexyarosh**
[See slack thread](https://metaboat.slack.com/archives/C013N8XL286/p1720732227609859)

### CODE REVIEW

**Comment by brunobergher** [comment]
Can we wait for @luizarakaki to review this too before merging?

File: docs/installation-and-operation/serialization.md
Comment (jeff-bruemmer) [inline docs/installation-and-operation/serialization.md:30]: This seems like a restatement of the first bullet, but I'm not opposed to including it if it helps people get the point.

And should it be "between" or "to"? Does "between" imply bi-directionality?

```suggestion
- **Duplicating assets to other Metabase instances**. Export the "template" data from a source Metabase and import them to one or more target instances.
```

File: docs/installation-and-operation/serialization.md
Comment (jeff-bruemmer) [inline docs/installation-and-operation/serialization.md:37]: We should emphasize "within", as that's the key distinction:

```suggestion
> Serialization isn't intended for use cases like duplicating assets or swapping data sources _within_ the same Metabase instance. If you're using serialization for duplicating assets within the same instance, check out [How export works](#how-export-works), [How import works](#how-import-works), and the directions for your use case in [Other uses of serialization](#other-uses-of-serialization)
```

File: docs/installation-and-operation/serialization.md
Comment (jeff-bruemmer) [inline docs/installation-and-operation/serialization.md:660]:
```suggestion
Serialization is intended for version control, staging-to-production workflows, and duplicating assets to other Metabase instances. While it's possible to use serialization for other use cases (like duplicating assets _within_ a single instance), we don't officially support these use cases.
```

File: docs/installation-and-operation/serialization.md
Comment (jeff-bruemmer) [inline docs/installation-and-operation/serialization.md:662]: "Unsupported" is more precise than "alternative".

```suggestion
We're providing some directions on how to approach these unsupported use cases, but you should use them at your own risk. We strongly recommend that you test any process involving serialization on a non-production instance first, and reach out to [help@metabase.com](mailto:help@metabase.com) if you have any questions.
```

**Review by luizarakaki — APPROVED** [review]

**Pull request merged**
